helm/values.yaml

```diff
@@ -95,6 +95,22 @@ controller:
   extraEnv: {}
   envFrom: []
 
+  serviceAccount:
+    create: true
```

Comment:
do we want to create by default?

Reply:
yes, I think this is desirable. it's mostly standard practice to not use the default service account when releasing a chart